src/pages/Home.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import '../assets/App.css';
+import '../assets/cars.css';
+import Cars from '../cars.json';
+
+const Home = () => (
+  <div className="content">
+    <div className="cars">
+      {Cars.map((car) => (
+        <div className="car-container" key={car.id}>
+          <div className="car-image">
+            <span className="dot" />
+            <img src={car.image} alt={car.model} />
+          </div>
+          <h4>{car.model}</h4>
+          <p>{car.description}</p>
```

Comment:
Please kindly add missing parameters like "color" and "released_date" where we wanna display. Even if it is not decided yet, just a low-fidelity wireframe display is enough.

Reply:
Done ✅